src/pages/TCY/components/Unstake/UnstakeInput.tsx

```diff
@@ -0,0 +1,270 @@
+    // TODO(gomes): revert me after tcy launch
+    // bnOrZero(amountCryptoPrecision).isZero() ||
+    isEstimatedFeesDataError ||
+    !amountCryptoPrecision ||
+    !fiatAmount
+
+  const confirmCopy = useMemo(() => {
+    if (errors.amountCryptoPrecision) return errors.amountCryptoPrecision.message
+    return translate('defi.unstake')
+  }, [errors.amountCryptoPrecision, translate])
+
+  useEffect(() => {
+    setValue('accountId', accountId ?? '')
+  }, [accountId, setValue])
+
+  return (
+    <Stack>
+      {headerComponent}
+      <FormControl isInvalid={Boolean(errors.amountCryptoPrecision)}>
+        <TradeAssetInput
+          assetId={selectedStakingAsset?.assetId ?? ''}
+          assetSymbol={selectedStakingAsset?.symbol ?? ''}
+          assetIcon={selectedStakingAsset?.icon ?? ''}
+          onAccountIdChange={noop}
+          label={translate('common.amount')}
+          isAccountSelectionDisabled
+          placeholder={translate('common.enterAmount')}
+          onToggleIsFiat={toggleIsFiat}
+          onChange={handleAmountChange}
+          isFiat={fieldName === 'fiatAmount'}
+          cryptoAmount={amountCryptoPrecision}
+          fiatAmount={fiatAmount}
+          isReadOnly
+          formControlProps={formControlProps}
+          rightComponent={ReadOnlyAsset}
+          // eslint-disable-next-line react-memo/require-usememo
+          percentOptions={[]}
+        >
+          <Stack spacing={4} px={6} pb={4}>
+            <Slider value={unstakePercent} onChange={handleUnstakePercentChange}>
+              <SliderTrack>
+                <SliderFilledTrack bg='blue.500' />
+              </SliderTrack>
+              <Tooltip label={`${unstakePercent}%`}>
+                <SliderThumb boxSize={4} />
+              </Tooltip>
+            </Slider>
+            <Flex width='full' justifyContent='space-between' fontSize='xs' color='text.subtle'>
+              <Amount.Fiat value={0} />
+              <Amount.Fiat
+                value={tcyStaker?.amount ? fromBaseUnit(tcyStaker.amount, THOR_PRECISION) : '0'}
+              />
+            </Flex>
+          </Stack>
+        </TradeAssetInput>
+      </FormControl>
+      <Card>
+        <CardFooter
+          flexDirection='column'
+          gap={4}
+          bg='background.surface.raised.base'
+          borderBottomRadius='xl'
+        >
+          <Row fontSize='sm' Tooltipbody={tooltipBody}>
+            <Row.Label>{translate('trade.networkFee')}</Row.Label>
+            <Row.Value>
+              <Skeleton isLoaded={!!estimatedFeesData}>
+                <Amount.Fiat value={estimatedFeesData?.txFeeFiat ?? 0} />
+              </Skeleton>
+            </Row.Value>
+          </Row>
+          <Button
+            colorScheme={isValid ? 'blue' : 'red'}
+            size='lg'
+            width='full'
+            onClick={handleUnstake}
+            isDisabled={isDisabled}
+            isLoading={isEstimatedFeesDataLoading}
+          >
+            {confirmCopy}
```

Comment:
As with the Stake PR, we probably want to handle no wallet connected. Probably a follow-up PR to tackle both is in order!